src/util/CustomButton.js

```diff
@@ -6,10 +6,23 @@ import Tooltip from '@material-ui/core/Tooltip';
 // Icons
 import IconButton from '@material-ui/core/IconButton';
 
-export default ({ children, onClick, tip, btnClassName, tipClassName }) => (
+export default ({
+  children,
+  onClick,
+  tip,
+  btnClassName,
+  tipClassName,
+  disabled,
+}) => (
   <Tooltip title={tip} className={tipClassName} placement='top'>
-    <IconButton onClick={onClick} className={btnClassName}>
-      {children}
-    </IconButton>
+    <span>
```

Comment:
Why is there a span here?

Reply:
Material-ui originally threw an error saying that it needs the span, but now I can't seem to repeat the error. Not sure if it's needed or not.

Reply:
![image](https://user-images.githubusercontent.com/7955498/70953037-f7653680-201c-11ea-906b-2199b88ad695.png)

Nevermind, I got it to repeat. Without the span, it throws an error that the tooltip is confused by the disabled command so, there needs to be a span to circumnavigate the issue.

Reply:
Interesting, I guess this is a material-ui implementation specific thing.